.github/workflows/mcp-javascript-cd.yml

```
@@ -27,12 +27,10 @@ jobs:
         with:
           node-version: 24
           cache: pnpm
-          cache-dependency-path: "mcp-server/pnpm-lock.yaml"
-          registry-url: "https://registry.npmjs.org"
+                    registry-url: "https://registry.npmjs.org"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)